src/ActionList/Item.tsx

```diff
@@ -59,9 +60,50 @@ export interface ItemProps extends React.ComponentPropsWithoutRef<'div'>, SxProp
    * Designates the group that an item belongs to.
    */
   groupId?: string
+
+  /**
+   * Items that are disabled can not be clicked, selected, or navigated through.
+   */
+  disabled?: boolean
+
+  /**
+   * Callback that will trigger both on click selection and keyboard selection.
+   */
+  onAction?: (item: Partial<ItemProps>, event?: MouseEvent | KeyboardEvent) => void
```

Comment:
CI is failing right now because it's not liking the event type here. Any thoughts on what `event` needs to be?

Reply:
Try `React.MouseEvent | React.KeyboardEvent`. These are the react "synthetic" versions of these event objects, which are generated when you listen with one of the `on{action}` event handlers on a JSX element.

Another fix, which might be preferable, is to keep the `MouseEvent | KeyboardEvent`, but when you call `onAction` you pass `event.nativeEvent` instead of just `event`.